DataAccessLayer/Crud/CategoryCrud.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using DataAccessLayer.DataBaseModels;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.Crud
+{
+    public class CategoryCrud 
+    {
+        /// <summary>
+        /// Get all categories from table DB
+        /// </summary>
+        /// <returns>List of categories</returns>
+        public List<Category> GetAll()
+        {
+            try
+            {
+                using (LanguageSkillsDBContext db = new LanguageSkillsDBContext())
+                {
+                    return db.Categories.ToList();
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine($"\nClassName - {this.GetType().Name}\nMethodName - {MethodBase.GetCurrentMethod().Name}" +
+                                  $"\nException message: {e.Message}. \nException stack trace: {e.StackTrace}");
+                return new List<Category>();
+            }
+        }
+
+        /// <summary>
+        /// Get category by Id from table DB
+        /// </summary>
+        /// <param name="id">Category Id</param>
+        /// <returns>Category</returns>
+        public Category GetById(int id)
+        {
+            try
+            {
+                using (LanguageSkillsDBContext db = new LanguageSkillsDBContext())
+                {
+                    return db.Categories.Find(id);
```

Comment:
How this method can work?
According to documentation - it takes predicate as param:
https://docs.microsoft.com/ru-ru/dotnet/api/system.collections.generic.list-1.find?view=netcore-3.1